tests/acceptance/github-release-test.js

```diff
@@ -0,0 +1,99 @@
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+import startApp from 'dummy/tests/helpers/start-app';
+import Pretender from 'pretender';
+import Ember from 'ember';
+
+const { run } = Ember;
+
+let server, app, container, store;
+
+moduleForAcceptance('Acceptance | github release', {
+  beforeEach() {
+    server = new Pretender();
+    server.prepareBody = function (body) {
+      return JSON.stringify(body);
+    };
+    app = startApp();
+    container = app.__container__;
+    store = run(container, 'lookup', 'service:store');
+  },
+
+  afterEach() {
+    server.shutdown();
+    run(app, app.destroy);
+    Ember.BOOTED = false;
+  }
+});
+
+test('finding a release without authorization', function (assert) {
```

Comment:
Can you add some `assert.expect` to tests that run async assertions? That will help us make sure the assertions are actually run, in case something breaks.

These tests look great otherwise!